v3/plugins/cdr/code-server/latest

```diff
@@ -0,0 +1 @@
+2.1523-vsc1.38.1
```

Comment:
The way latest currently works is that you need folders with a copy of the latest meta.yaml (I know, it's a hack-y solution). 

For selecting `latest` in the Che UI to work, this should be
- File: `v3/plugins/cdr/code-server/latest/meta.yaml`
- Content: Copy of `/v3/plugins/cdr/code-server/2.1523-vsc1.38.1/meta.yaml`, with `.version=latest`.